.gitignore

```
@@ -2,3 +2,5 @@ docker/mysql/data
 phpmyadmin
 .env
 .vscode
+auction/
+!auction/.gitkeep
```

これだとあらゆるパスに存在する `auction` フォルダに対して設定が効いてしまいます
詳しいパスを指定しましょう！


```suggestion
html/mycakeapp/webroot/img/auction/*
!html/mycakeapp/webroot/img/auction/.gitkeep
```